kpatch-build/kpatch-build

```diff
@@ -65,6 +65,15 @@ LLD="${CROSS_COMPILE:-}ld.lld"
 READELF="${CROSS_COMPILE:-}readelf"
 OBJCOPY="${CROSS_COMPILE:-}objcopy"
 
+SUPPORTED_DISTRO="
```

Comment:
Please make it plural: SUPPORTED_DISTROS.  Or even better, SUPPORTED_RPM_DISTROS.

Reply:
Thank you for your patient review, I have move this variable to SUPPORTED_RPM_DISTROS.